Octokit.Tests/Clients/DeploymentsClientTests.cs

```diff
@@ -94,11 +119,11 @@ public void PostsToDeploymentsUrl()
         {
             var connection = Substitute.For<IApiConnection>();
             var client = new DeploymentsClient(connection);
-            var expectedUrl = "repos/owner/name/deployments";
+            var expectedUrl = ApiUrls.Deployments("owner", "name");
```

Comment:
Don't change this to call the same code that the app does - let's keep this as-is to verify we're also formatting the URL correctly...